mypyc/irbuild/ll_builder.py

```diff
@@ -891,6 +891,9 @@ def make_dict(self, key_value_pairs: Sequence[DictEntry], line: int) -> Value:
 
         return result
 
+    def new_list_op_with_length(self, length: Value, line: int) -> Value:
+        return self.call_c(new_list_op, [length], line)
```

Comment:
Add docstring. Document that the new list is uninitialized and needs to be initialized with `CPyList_SetItemUnsafe` ops. Also document the expected rtype of `length`.